api.yaml

```diff
@@ -133,8 +170,26 @@ endpoints:
             integration.request.querystring.limit: method.request.querystring.limit
             integration.request.querystring.offset: method.request.querystring.offset
             integration.request.querystring.page: method.request.querystring.page
+            integration.request.querystring.page_size: method.request.querystring.page_size
           responses:
             200:
+              responseTemplates:
+                application/json: |
+                    #set($inputRoot = $input.path('$')) #set($host = $stageVariables.gateway_host)
+                    #macro (updateURL $url)
```

Comment:
@brittneyexline unless I'm mistaken, this file is generated from `api-compact.yaml` so there's some repetition from the expansion. Any code-quality questions should focus on the compact variant. (The expanded version is required for gateway integration, the compact version is for developers.)